x/ccv/provider/types/keys.go

```diff
@@ -158,6 +158,23 @@ const (
 	// that corresponds to the N% of the top validators that have to validate this consumer chain
 	TopNBytePrefix
 
+	// ValidatorsPowerCapPrefix is the byte prefix storing the mapping from a consumer chain to the power-cap value of this chain,
+	// that corresponds to p% such that no validator can have more than p% of the voting power on the consumer chain.
+	// Operates on a best-effort basis.
+	ValidatorsPowerCapPrefix
```

Comment:
Do not add keys before other keys (Marius left a comment on the main PR to this effect)

Reply:
I agree with Marius' [comment](https://github.com/cosmos/interchain-security/pull/1809#discussion_r1577712376) and your [change](https://github.com/cosmos/interchain-security/commit/7b9842f6fe51c76c0c7f00b62d6264c86cb023e9) because there the ordering was messing with `ConsumerValidatorBytePrefix` that already exists in v16.

In this case however, both `ConsumerRewardsAllocationBytePrefix` and `ConsumerCommissionRatePrefix` are added as part of the [PSS work](https://github.com/cosmos/interchain-security/pull/1809), so I do not see any potential issue with this. Before PSS launches, we can reorder those PSS-related prefixes as we see fit. It seems to me that having the power cap, validator-set cap, etc. fit well under the `TopNBytePrefix` because they are all characteristics of a consumer chain.

Reply:
Good point, thanks for double-checking!